README.md

```diff
@@ -0,0 +1,75 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
```

Comment:
- [ ] Kindly replace the information on the README template with the information on your project. 👇🏽 

> Please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

- HTML & CSS could be an option.